src/index.d.ts

```diff
@@ -441,6 +441,11 @@ export interface RequiredOptions extends doc.printer.Options {
    * @deprecated use bracketSameLine instead
    */
   jsxBracketSameLine?: boolean;
+  /**
+   * Where to print operators when binary expressions wrap lines.
+   * @default "end"
+   */
+  experimentalOperatorPosition?: "start" | "end";
```

Comment:
Looking at other typings, I guess we do not need `?` here. This type is named `RequiredOptions` so all values are mandatory.

Since we need to make a change anyway, let’s move `experimentalOperatorPosition` before `experimentalTernaries` (to keep them closer together in alphabetical order).

Reply:
Yes, no need for question mark, removed and re-ordered. 

Thanks.